salvo/src/lib/api/env.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package salvo;
+
+// Capture all Environment variables required for the benchmark
+message EnvironmentVars {
+  // Specify the IP version for tests
+  oneof envoy_ip_test_versions {
+    bool v4only = 1;
```

Comment:
For code readability consider longer names, since only the field name will appear in the code:

Example:
`ipv4_only`, `ipv6_only`, `dual_stack`

Or similar. Currently seeing the name `all` in code won't give much context.